LayoutTests/TestExpectations

```diff
@@ -6003,3 +6003,7 @@ js/ShadowRealm-iframe-detach.html [ Pass Failure ]
 js/ShadowRealm-iframe-sandboxed.html [ Pass Failure ]
 js/ShadowRealm-importValue.html [ Pass Failure ]
 js/ShadowRealm-worker.html [ Pass Failure ]
+
+# Old SVG tests with invalid results in SVG2 + Color 4
+imported/w3c/web-platform-tests/svg/import/color-prop-05-t-manual.svg [ Pass Failure ]
+svg/W3C-SVG-1.1-SE/color-prop-05-t.svg [ Pass Failure ]
```

Comment:
This patch should not contain any functional changes and so shouldn't affect any tests.